biome.jsonc

```diff
@@ -24,7 +24,6 @@
 			// --- ignore runtime code in browser
 			"packages/rspack/hot",
 			"packages/rspack-dev-server/client",
-			"packages/rspack/src/builtin-plugin/css-extract/hmr",
```

Comment:
these files has been migrated to typescript, and it would be transformed by loader, so it is okay